scripts/javasc.js

```diff
@@ -26,21 +26,118 @@ function userFeedback(userResponse, correctAnswer){
         return 0;
     }
 }
+ 
+function iterateQuestions(questions, answers, acceptedAnswers){
+    // Main loop to cycle through the questions
+    for (let i = 0; i < questions.length; i++) {
+        let ans = prompt(questions[i]);
+        while (!inputChecker(ans, acceptedAnswers)) {
+            alert('Please provide a valid input y/n/yes/no');
+            ans = prompt(questions[i]);
+        }
+
+        numberOfCorrect += userFeedback(ans, answers[i]);
+    } 
+}
+
+function guessMyNumber(numRange, numOfGuessesNum){

```

Comment:
I also really like that this takes in these arguments, to change the guessing game however you want!